### PR TITLE
feat add zookeeper

### PR DESCRIPTION
**Pull request by lachie83**
Helm Chart for https://github.com/kubernetes/contrib/tree/master/pets/zookeeper

### CODE REVIEW

**Comment by viglesiasce** [comment]
@lachie83 would you mind pulling in the last changes from master (reorg) then putting this chart in the incubator directory?

**Comment by lachie83** [comment]
@viglesiasce sure thing.

**Comment by viglesiasce** [comment]
Thanks @lachie83 !!

**Comment by lachie83** [comment]
@viglesiasce - Moved to incubator dir

**Comment by viglesiasce** [comment]
ok to test

**Comment by viglesiasce** [comment]
ok to test

**Comment by viglesiasce** [comment]
ok to test

**Comment by viglesiasce** [comment]
retest this please

**Comment by viglesiasce** [comment]
@k8s-bot test this

**Comment by viglesiasce** [comment]
@k8s-bot test this

File: incubator/zookeeper/README.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+# Deep dive
+
+
+## Failover
+
+You can test failover by killing the leader. Insert a key:
+```console
+$ kubectl exec zoo-0 -- /opt/zookeeper/bin/zkCli.sh create /foo bar;
```
Comment (viglesiasce) [inline incubator/zookeeper/README.md]: Then names of the pods are actually $RELEASE-NAME-zoo-0

**Comment by viglesiasce** [comment]
@lachie83 one nit in the README. Tested it and worked as expected otherwise.

**Comment by lachie83** [comment]
Thanks @viglesiasce. Updated README

**Comment by viglesiasce** [comment]
Thanks @lachie83!

**Pull request merged**